toolkit/tools/imager/imager.go

```diff
@@ -481,6 +481,12 @@ func buildImage(mountPointMap, mountPointToFsTypeMap, mountPointToMountArgsMap m
 		setupChrootPackages = append(setupChrootPackages, toolingPackage.Name)
 	}
 
+	logger.Log.Infof("Hidepid is %v.", systemConfig.HidePid)
+	hidepidflag := false
```

Comment:
Should we enable hidepid by default and let the user opt out? It seems like a good security feature to have enabled by default in our images.